src/regressionmodel.jl

```diff
@@ -117,3 +117,27 @@ Return the offset used in the model, i.e. the term added to the linear predictor
 known coefficient 1.
 """
 function offset end
+
+"""
+    linearpredictor(model::RegressionModel)
+
+Return the model's linear predictor, `Xβ` where `X` is the model matrix and `β` is the
+vector of coefficients, or `Xβ + offset` if the model was fit with an offset.
+"""
+function linearpredictor(model::RegressionModel)
+    X = modelmatrix(model)
+    β = coef(model)
+    η = X * β
+    off = offset(model)
+    if off !== nothing && !isempty(off)
+        η .+= off
+    end
+    return η
+end
+
+"""
+    linearpredictor!
+
+In-place version of [`linearpredictor`](@ref).
```

Comment:
Please specify the expected arguments.

Reply:
I was following suit with `predict!` and `reconstruct!`, neither of which specify the arguments.

Reply:
Specifically, their docstrings are, respectively,
```
"""
    predict!

In-place version of [`predict`](@ref).
"""
```
and
```
"""
    reconstruct!

In-place version of [`reconstruct`](@ref).
"""
```

Reply:
Yes, but that's precisely something that https://github.com/JuliaStats/StatsAPI.jl/issues/18 mentioned as problems. :-)

Reply:
Okay, added more info.